libs/application/templates/parental-leave/src/fields/PeriodPercentage/PeriodPercentage.tsx

```diff
@@ -133,6 +171,15 @@ export const PeriodPercentage: FC<PeriodPercentageField> = ({
           value={StartDateOptions.SPECIFIC_DATE}
         />
       )}
+
+      {isUsingAllRemainingDays && (
+        <input
+          type="hidden"
+          ref={register}
+          name={`periods[${currentIndex}].daysToUse`}
+          value={remainingRights}
+        />
+      )}
```

Comment:
Looks like it doesn't clear an existing value. Would be nice if this step could support edits/back even though it doesn't happen now.